src/QMCWaveFunctions/Jastrow/J2OMPTarget.cpp

```diff
@@ -191,7 +225,7 @@ template<typename FT>
 void J2OMPTarget<FT>::copyFromBuffer(ParticleSet& P, WFBufferType& buf)
```

Comment:
The is the legacy driver thing. Won't touch.

Reply:
So this will get deleted along with the legacy driver?  Can we mark it deprecated, it shouldn't be used for new code.

Reply:
Since we are testing legacy drivers with J2OMPTarget which is default when ENABLE_OFFLOAD=ON. they are still in use.
If we mark deprecated, the amount of warning will be massive so I don't want to see them.
They should not be called when batched driver is used.

Reply:
I recall we've discussed the explosion of warnings from use of the c++ [[deprecated]] attribute and I agree we should avoid it.
I'll look into the `\deprecated` doxygen tag.